pyproject.toml

```diff
@@ -23,22 +23,23 @@ dependencies = [
     "pyserial",
     "huggingface-hub==0.34.4",
     "sounddevice==0.5.1",
-    "soundfile==0.13.1",    
+    "soundfile==0.13.1",
     "reachy-mini-rust-kinematics>=1.0.1",
     "asgiref",
     "aiohttp",
     "log-throttling==0.0.3",
     "pyusb>=1.2.1",
     "libusb_package>=1.0.26.3",
     "pip>=25",
+    "rerun-sdk==0.27.1",
```

Comment:
Could you leave rerun dependency in the optional dev section? We'd like to keep the core python package as small as possible.